lighthouse-core/report/v2/renderer/category-renderer.js

```diff
@@ -116,6 +116,99 @@ class CategoryRenderer {
   }
 
   /**
+   * @param {!ReportRenderer.AuditJSON} audit
+   * @param {number} scale
+   * @return {!Element}
+   */
+  _renderPerfHintAudit(audit, scale) {
+    const extendedInfo = /** @type {!CategoryRenderer.PerfHintExtendedInfo}
```

Comment:
yeah, my concern is breaking the seal on `extendedInfo` :/ These things tend to grow with time (see: `extendedInfo` itself). It also breaks the `details`->rendering info, `extendedInfo` -> debug/for other tools info split. Really it would be better even if it was exactly the same but called basically anything else :)